adds styling to homepage

Lily-Brown opened the pull request.

Lily-Brown committed:
adds styling to homepage

teddycoleman commented:
Awesome - pulled it in!

I added models for the stop and tourstop.  Also added get method for pulling all stops for a given tour.

Lily-Brown commented:
Sweet! We're killing it so far!

Lily

> On Sep 28, 2016, at 10:35 PM, teddycoleman notifications@github.com wrote:
> 
> Awesome - pulled it in!
> 
> I added models for the stop and tourstop. Also added get method for pulling all stops for a given tour.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
